src/py42/choices.py

```diff
@@ -0,0 +1,14 @@
+from py42.util import get_attribute_keys_from_class
+
+
+class Choices:
+    """Helper class to provide the choices() method to a class."""
+
+    @classmethod
+    def choices(cls):
+        """Returns attribute names for the given class.
```

Comment:
Hmm, one thing I noticed (this has always been the case, it's not due to this change), is that this description is incorrect. `.choices()` returns attribute _values_ not names (or "keys", based on the naming of `get_attribute_keys_from_class()` func.

Example, this is what .choices() currently returns:
```
>>> EventTimestamp.choices()
['PT15M', 'P14D', 'P1D', 'PT1H', 'P7D', 'P30D', 'P3D', 'PT3H', 'PT12H']
```
if it were actually grabbing attribute names it would be:

```
>>> EventTimestamp.choices()
['FIFTEEN_MINUTES', 'FOURTEEN_DAYS', 'ONE_DAY', 'ONE_HOUR', 'SEVEN_DAYS', 'THIRTY_DAYS', 'THREE_DAYS', 'THREE_HOURS', 'TWELVE_HOURS']
```

@alanag13 @unparalleled-js what was the original intent of the choices() usage? Names or values?

Reply:
Good catch -- values, the intent is to enumerate all options for a certain value that could be sent to the API. The name of the python var doesn't matter. We should update the docs accordingly.

Reply:
@tora-kozic if we change this docstr to attribute _values_ this all LGTM.

Reply:
I can do that! Would it be also appropriate to rename `get_attribute_keys_from_class()` to `get_attribute_values_from_class()` or do we want to leave that as is?